snapcraft.yaml

```diff
@@ -0,0 +1,38 @@
+name: zeal # you probably want to 'snapcraft register <name>'
+version: '0.4.0' # just for humans, typically '1.2+git' or '1.3.2'
+summary: Zeal is a simple offline documentation browser inspired by Dash.
+description: |
+        Zeal is a simple offline documentation browser inspired by Dash.
+
+grade: stable
+confinement: strict
+apps:
+  zeal:
+    command: desktop-launch $SNAP/bin/zeal
+    plugs: [network, x11, unity7, browser-support]
+parts:
+
+  zeal:
+    plugin: cmake
+    source: https://github.com/zealdocs/zeal/archive/v$SNAPCRAFT_PROJECT_VERSION.tar.gz
+    build-packages:
+      - build-essential
+      - libarchive-dev
+      - libappindicator-dev
+      - libxcb-keysyms1-dev
+      - zlib1g-dev
```

Comment:
Do all dependencies need to be listed explicitly? I am pretty sure some of these depend on others already.